api/organisations/models.py

```diff
@@ -262,7 +259,7 @@ def update_plan(self, plan_id):  # type: ignore[no-untyped-def]
 
     @property
     def can_auto_upgrade_seats(self) -> bool:
-        return self.plan in settings.AUTO_SEAT_UPGRADE_PLANS
+        return self.subscription_plan_family == SubscriptionPlanFamily.SCALE_UP
```

Comment:
Just to future proof this, I wonder if we should add a check on SaaS? 

```suggestion
        return is_saas() and self.subscription_plan_family == SubscriptionPlanFamily.SCALE_UP
```

I don't see a scenario in which scale up is used in any environment other than SaaS, but perhaps worth belt and braces approach?

Reply:
> future proof

Surely we'll have to modify more that this code path if we decide to charge by seats on enterprise?

Reply:
Explained offline, resolving this (basically, it's possible to trigger this code path for self-hosted installations by assigning a scale-up plan to the org subscription).